orbs/shared/jobs/merge.yaml

```diff
@@ -0,0 +1,31 @@
+description: Merges the base (to merge into) branch into the head (current) branch and pushes the result
+executor:
+  name: testbed-docker
+docker:
+  - image: gcr.io/outreach-docker/bootstrap/ci:stable
+    auth:
+      username: _json_key
+      password: $GCLOUD_SERVICE_ACCOUNT
+parameters:
+  head:
+    type: string
+  base:
+    type: string
```

Comment:
```suggestion
  head:
    description: The current branch, this will have base merged into it. This is only supplied to ensure we're on the correct branch when merging.
    type: string
  base:
    description: The branch to merge into head
    type: string
```